<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <item>
     <widget class="QFrame" name="drop_shadow_frame">
      <property name="styleSheet">
       <string notr="true">QFrame{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(40, 43, 75, 255));
color:rgb(100, 194, 202);
border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_title">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>381</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Uroob</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: none;</string>
       </property>
       <property name="text">
        <string>System Monitor</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_description">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>381</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: none;
color:rgb(150, 150, 150)</string>
       </property>
       <property name="text">
        <string>Zinus Corp.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>321</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
background-color: rgb(120, 120, 124);
border-style: none;
color: rgb(220, 220, 220);
border-radius: 8px;
text-align: center;
}

QProgressBar::chunk {
border-radius: 8px;
background-color: qlineargradient(spread:pad, x1:0, y1:0.494, x2:1, y2:0.488636, stop:0 rgba(52, 153, 216, 255), stop:1 rgba(85, 255, 127, 255));
}</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_loading">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>140</y>
         <width>381</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: none;
color:rgb(150, 150, 150)</string>
       </property>
       <property name="text">
        <string>Loading...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_credit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>361</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: none;
color:rgb(150, 150, 150)</string>
       </property>
       <property name="text">
        <string>By: Oussama Talaoui</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
